javascript/ql/src/semmle/javascript/dataflow/TaintTracking.qll

```diff
@@ -827,6 +827,28 @@ module TaintTracking {
     override predicate appliesTo(Configuration cfg) { any() }
   }
 
+  /** A check of the form `type x === "undefined`, which sanitized `x` in its "then" branch. */
```

Comment:
Missing a `"` at the end here: type `type x === "undefined"` vs `type x === "undefined`